### DISCUSSION TITLE
Compatibility with Excel

### DISCUSSION THREAD

**Discussion by talestomaz** [goosepirate/lox365#14] [General]
Hi, in my LibreOffice installation, the XLOOKUP function of Lox365 works, but is not compatible with MS Office. When I have to access the file via OneDrive, for example, it substitutes the name of the function for COM.GOOSEPIRATE.LOX365.OXT.XLOOKUP and returns no valid result. Is this a bug or a feature? Thanks!

**Comment by goosepirate** [comment]
Hi. This is intended. Lox365 works with LibreOffice Calc only. It does not work with Excel.

**Reply by emilzak** [reply]
I have very strange behavior when testing this exact case. I generated a file using Microsoft Excel, which has a simple table and a XLOOKUP. Opening this excel using LibreOffice on different platforms gives me different results.

    MacOS, LibreOffice 7.4.3 **MS-Excel created file, XLOOKUP**  - Doesn't work
    Ubintu Linux, LibreOffice 7.3.7, **MS-Excel created file, XLOOKUP**  - Works!
    Debian 12 Linux, LibreOffice 24xxx, **MS-Excel created file, XLOOKUP**  - Doesn't work

What makes LibreOffice recognize XLOOKUP perfectly on Ubuntu without "COM.GOOSEPIRATE.LOX365.OXT." ?

**Reply by emilzak** [reply]
Update:
It looks like 7.3.7 work always works when other versions do not...
